vertical-pod-autoscaler/e2e/v1/common.go

```diff
@@ -321,6 +320,21 @@ func SetupVPAForNHamsters(f *framework.Framework, n int, cpu string, mode vpa_ty
 	}
 
 	InstallVPA(f, vpaCRD)
+	return vpaCRD
+}
+
+// PatchVpaUpdatePolicy modifies existing VPA object 'hamster-vpa' with new UpdatePolicy.
+func PatchVpaUpdatePolicy(f *framework.Framework, vpa *vpa_types.VerticalPodAutoscaler, policy *vpa_types.PodUpdatePolicy) {
+	newSpec := vpa.Spec.DeepCopy()
+	newSpec.UpdatePolicy = policy
+	bytes, err := json.Marshal([]patchRecord{{
+		Op:    "replace",
+		Path:  "/spec",
+		Value: *newSpec,
+	}})
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
```

Comment:
Please provide a short description to make debugging this test easier if it fails. Something like "when marshalling VPA UpdatePolicy patch" will be enough

https://pkg.go.dev/github.com/onsi/gomega#Assertion

Reply:
Removed this helper completely.